build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+	id 'org.springframework.boot' version '2.4.3'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+}
+
+group = 'by.techart'
```

Comment:
I think it would be better to use an existing domain name such as `itechart.com` which allows us to identify a company

Reply:
Okay, I will change it.